sansio_lsp_client/client.py

```diff
@@ -1,10 +1,13 @@
+import os
 import enum
 import pprint
 import typing as t
+import logging
```

Comment:
Here's something that's 90% personal preference: do we want to go with `logging` as a log library?

I am a *big* proponent of [structlog](https://www.structlog.org/en/stable/), a structured logging library.

Especially since this is a library to be used in editors, being able to correctly categorize log messages and display them to the user in a coherent way if needed could be important.

Let me know what you think @Akuli

Reply:
I expect libraries to use `logging`. In Porcupine, and in other projects, I just configure logging and everything goes to the same log file.

Apparently structlog is compatible with logging, and so it won't break backwards compatibility if we change to it later.